spec/controllers/users_controller_spec.rb

```diff
@@ -2,9 +2,9 @@
 require 'support/login_user'
 
 RSpec.describe UsersController, type: :controller do
-  include_context 'login user'
   
   describe 'GET auto_complete' do
+    include_context 'login user'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.